src/core/reducers/version.js

```diff
@@ -0,0 +1,26 @@
+import {
+  VERSION_GET,
+  VERSION_LOADED,
+  VERSION_FAILED,
+} from 'core/constants';
+
+const initialState = {
+  loading: false,
+};
+
+export default function version(state = initialState, action) {
+  const { payload } = action;
+  switch (action.type) {
+    case VERSION_GET:
+      return { ...state, ...payload, loading: true };
+    case VERSION_LOADED:
+      return { ...state, ...payload.result, loading: false };
```

Comment:
same thing: I think this needs `error: false` in case the last state was an error state